src/main/java/io/pinecone/clients/Pinecone.java

```diff
@@ -1023,6 +1025,16 @@ public Builder withProxy(String proxyHost, int proxyPort) {
             return this;
         }
 
+        public Builder withHost(String host) {
+            this.host = host;
+            return this;
+        }
+
+        public Builder withTlsEnabled(boolean enableTls) {
+            this.enableTls = enableTls;
+            return this;
+        }
+
```

Comment:
Should these have some sort of documentation similar to `withProxy` above?

Reply:
Yes I need to the docstrings